src/plugins/poster/poster.js

```diff
@@ -42,12 +42,8 @@ export default class PosterPlugin extends UIContainerPlugin {
     this.listenTo(this.container, Events.CONTAINER_STOP, this.onStop)
     this.listenTo(this.container, Events.CONTAINER_PLAY, this.onPlay)
     this.listenTo(this.container, Events.CONTAINER_ENDED, this.onStop)
-    if (this.container.buffering) {
-      process.nextTick(() => this.onBuffering())
```

Comment:
Did you test with FlasHLS? ( I think this `process.nextTick` is to handle flash)